### ISSUE TITLE
Pokemons won't appear since last login server downtime

### ISSUE THREAD

**Issue by pokefurydotcom** [AHAAAAAAA/PokemonGo-Map#3406]
Since the last login server downtime I got no pokemons showing up

## Expected Behavior
Pokemons appearing on the map

## Current Behavior
Pokemons not appearing on the map

## Steps to Reproduce (for bugs)

1. python /home/pokefury/PokemonGo-Map/runserver.py -u xxx -p xxx -st 5 -k xxx -l "xxx" -ns -t 1 -fl -sd 5

## Your Environment
* Version used: Current main
* Environment name and version (e.g. Python 2.7): Python 2.7 / Ubuntu 14.04
* Operating System and version (desktop or mobile):  Both

**Comment by Ps-Pencil** [comment]
Same. Upserted 0 pokemon

**Comment by adityaAsh** [comment]
same dont know why :'(

**Comment by presidenten** [comment]
Same :(

**Comment by Whynick** [comment]
same :(

**Comment by licklake** [comment]
Same :(

**Comment by presidenten** [comment]
To be fair ... I dont really see any close by pokemons ingame either. Might be just a hiccup.

**Comment by Xavi1321** [comment]
Same :(

**Comment by FlarumHostingCo** [comment]
Yeap map now only show stops and gyms and that's being pulled from the database.  Hopefully not another PoGo API change on there end.

**Comment by pokefurydotcom** [comment]
They are working on making it live to brazil. That's either the api being overloaded or they found a way to block it. Let's hope it's not the latest.

**Comment by licklake** [comment]
how about we all shut our systems down, so they can go live in brazil and dont have to work on blocking us, until its fixed

**Comment by GamerBene19** [comment]
@licklake Good idea, mine is already down. Because when I noticed the server are going down (higher ping, etc.) I shut down my map.

**Comment by 1111121RicardoBrandao** [comment]
PTC login is having a lot of issues and after login im getting 0 in all values, they change something on their side probably

**Comment by snapper67** [comment]
Wouldn't a change on their side require a change to to their client app to match? My phone is still seeing objects, while the server is not.

**Comment by pokefurydotcom** [comment]
MapObjects in API changed from what I heard

**Comment by FlarumHostingCo** [comment]
The change might had been made in last patch and they just didn't go live with it on server side til today.   Hopefully though it is just an issue by Brazil going live today and will fix itself in a few hours.

**Comment by dese1ect** [comment]
The API changed. It will be fixed. No big deal.

**Comment by zachbadgett** [comment]
Users were forced updated to the new version recently, maybe it's just a header that needs to be sent?

**Comment by A5LAN** [comment]
any changes guys ? or still the same

**Comment by vinsilsim** [comment]
@A5LAN same!

**Comment by pokefurydotcom** [comment]
> pyr0ball: @everyone niantic has changed the API structure. Currently the map is not functional. Dev's are working on a fix now. Please be patient and avoid cluttering channels with unhelpful comments. Trolls will get kicked or banned

**Comment by vinsilsim** [comment]
@pokefurydotcom Thank you for the clarification.

**Comment by Invaderofbunnie** [comment]
Following :D

**Comment by TrackerTack** [comment]
Lets cheer the devs on for good luck in finding a fix to this endeavor

**Comment by itztq** [comment]
We are counting on you guys, lets hope for a stable fix!

**Comment by because-bot-fed** [comment]
Haha so I guess I just made 250 accounts for nothing. :|

**Comment by mvfsullivan** [comment]
Where can I go to follow updates?

**Comment by itztq** [comment]
@mvfsullivan  https://github.com/AHAAAAAAA/PokemonGo-Map/issues?q=is%3Aissue+is%3Aclosed

https://www.reddit.com/r/pokemongodev/comments/4w0mtr/getting_0_pokemon_upserted_in_last_few_minutes/

**Comment by cdhowie** [comment]
@mvfsullivan You can subscribe to this issue on GitHub, for example, which will email you whenever someone makes a comment.

However, this is only useful if the people making comments are *actually collaborating on a solution* by doing research or fixing code.  Posts like "me too," "when will this be fixed," "the devs are awesome," "niantic sucks," speculation about what changed, incorrect information about what changed, etc. (ironically including this comment) only serve to obfuscate and decrease the signal-to-noise ratio.

In other words, this is an issue tracker and **not** a discussion forum.

**Comment by andrewda** [comment]
I will also add to @cdhowie's comment that you should add a :+1: reaction to a comment if you are experiencing the same issue. This applies to all issues across all repositories and not just this one. I will start to be more proactive about removing "same", "me too", "niantic sucks", etc. comments.

**Comment by cmills47** [comment]
Can this be renamed to more succinctly address the issue? To me before submitting a dupe this looked like a login our server issue which this must definitely is not. After reading the whole thread and carefully parsing the title I can see it's technically correct but lots of people are confused and creating dipped and I think a rename would help.

Also, sorry for not knowing to search which would also find closed issues more appropriately named that link to this... Now I know and won't repeat that rookie mistake. Also, thanks to the people investing their valuable time and energy into this project and fixing issues so quickly... I'm impressed!

**Comment by kneel23** [comment]
Sigh I figured this would happen. Since the app update there were less Pokemon appearing on the map than actually existed in real life.  They must have slowly migrated over to the new "method" or whatever and now we are 100% SOL unless the script can be updated to pull from the new source *fingers crossed*

**Comment by davidblancop** [comment]
any news?

**Comment by FlarumHostingCo** [comment]
It's only been 3 hours @davidblancop

**Comment by gnu-user** [comment]
Thanks for putting in the effort to fix this, I was worried for a moment that they were severely throttling APIs or something worse.

**Comment by NoMotion** [comment]
If anyone is interested in following the progress or possibly participating I think most of the work is over in the [protobuf Git](https://github.com/AeonLucid/POGOProtos/issues/131)

**Comment by flightcoin** [comment]
Awesome, at least it is a known problem. I hope to see the updates soon!

**Comment by vinsilsim** [comment]
News?

**Comment by A5LAN** [comment]
still the same but they are working on it
